tests/integ/scala/test_function_suite.py

```diff
@@ -4484,29 +4484,67 @@ def test_iff(session, local_testing_mode):
         )
 
 
-@pytest.mark.skipif(
-    "config.getoption('local_testing_mode', default=False)",
-    reason="cume_dist is not yet supported in local testing mode.",
-)
 def test_cume_dist(session):
     Utils.check_answer(
         TestData.xyz(session).select(
-            cume_dist().over(Window.partition_by(col("X")).order_by(col("Y")))
+            "X", "Y", cume_dist().over(Window.partition_by(col("X")).order_by(col("Y")))
```

Comment:
I see some changes in the test, like adding "X", "Y" in the select statement, change sort from `False` to `True` in the check_answer.

what's the purpose of the change here? is the previous test not working in the implementation or it's for adding more test coverage here?

Reply:
thanks, now I understand why cols X and Y are introduced.

btw does the server guarantee the same ordering or can return different orders. wondering if we should mention this in the limitation or known gap section of local testing framework.

Reply:
I believe the server order is consistent, not sure if it has a guarentee.